data/json/items/ammo.json

```diff
@@ -285,9 +285,11 @@
         "ammo_type" : "BB",
         "damage" : 2,
         "range" : 12,
-        "dispersion" : 11,
-        "count" : 200,
-        "effects" : ["NEVER_MISFIRES"]
+        "dispersion" : 15,
```

Comment:
The innate dispersion of the BB ammo is now 10 more than the flintlock_ammo. The BB gun's dispersion could be increased from 150.